### ISSUE TITLE
Cannot use 'bunyan" as the custom logger.

### ISSUE THREAD

**Issue by ahowardsky** [balderdashy/captains-log#10]
The following part of code is suspicious:

		// `logger.log` or `logger.debug` exists.
52			if (!logger.log) {
53				throw new Error(
54					'Unsupported logger override!\n' +
55					'(has no `.log()` or `.debug()` method.)'
…	
59			// Fill in the gaps for the required log methods with
60			// reasonable guesses if the custom logger is missing any
61			// (only required method is `logger.log` or `logger.debug`)

It only checks whether the provided logger has '.log" function but did not check '.debug'.
According to the error message, it should have worked if the logger contains the standard "loger.{level}()" functions.

**Comment by sailsbot** [comment]
Thanks for posting, @ahowardsky.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
